frontend/appflowy_flutter/lib/plugins/database/widgets/cell/mobile_grid/mobile_grid_relation_cell.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:appflowy/mobile/presentation/bottom_sheet/show_mobile_bottom_sheet.dart';
+import 'package:appflowy/plugins/database/widgets/row/cells/cell_container.dart';
+import 'package:appflowy/plugins/database/application/cell/bloc/relation_cell_bloc.dart';
+import 'package:appflowy_popover/appflowy_popover.dart';
+import 'package:flowy_infra_ui/flowy_infra_ui.dart';
+import 'package:flutter/material.dart';
+
+import '../editable_cell_skeleton/relation.dart';
+
+class MobileGridRelationCellSkin extends IEditableRelationCellSkin {
```

Comment:
I guess it's better to hide it completely than show "Coming soon", what do you think?

Reply:
Relation isn't creatable on mobile yet